### DISCUSSION TITLE
Is there any active plan to support generation of a compilation database file for C++ projects?

### DISCUSSION THREAD

**Discussion by jbulow** [facebook/buck2#724] [Q&A]
PRs seems on this topic seems to be abandoned and broken. The developer experience of buck2 for C++ is very limited without this support.

**Comment by kdowney-lot49** [comment]
Have you taken a look at this 3rd-party project? I am currently trying to use it with VSCode, vcpkg, C++20 and buck2:

https://codeberg.org/Release-Candidate/Cxx-Buck2-vcpkg-Examples/src/branch/main/concat_compile_cmds

**Comment by alexlian** [comment]
PR #810 got since merged in. Noting for this discussion topic and will close.
